V22 imm deconv rerun and figure updates (#1458)
AlexsLemonade/OpenPBTA-analysis: analyses/immune-deconv/02-visualize_quantiseq.Rmd

         contains("cancer_group"),
         molecular_subtype)

> #### Array of groups we want to visualize (groups with N >= 15?), remove Other

Reviewer: Suggested rephrase - 
```suggestion
#### Determine the of groups we want to visualize from those with N>=15, excluding Other
```

# First, remove cd8_cd4_ratio
quantiseq_no_ratio <- quantiseq_subset %>%
  filter(cell_type != "cd8_cd4_ratio")

subtypes_celltypes_plot <- ggplot(quantiseq_no_ratio) +
  aes(x = molecular_subtype, y = score) + 
  geom_boxplot(outlier.shape = NA, color = "grey40", size = 0.2) + 

>   geom_jitter(shape = 1, width = 0.15, size = 0.75, alpha = 0.4, aes(color = cancer_group_hex)) + 

Reviewer: By having `shape=1` here but not also in the legend, we have a mismatch. I'd recommend removing `shape=1` from here (or you have to add `shape=1` to the `geom_point()` making the legend).

        axis.line = element_line(size = rel(0.7)),
        axis.ticks = element_line(size = rel(0.7)))

# Add legend
cg_color_df <- quantiseq_no_ratio %>%
  select(cancer_group_hex, cancer_group_display) %>%
  distinct()

cg_color_list <- cg_color_df$cancer_group_hex
names(cg_color_list) <- cg_color_df$cancer_group_display

legend_plot <- ggplot(quantiseq_no_ratio) + 
  aes(x = score, y = score, color = cancer_group_display) + 
  geom_point() +
  scale_color_manual(name = "Cancer group", values = cg_color_list) + 

>   theme_pubr()

Reviewer: I'm realizing this legend is way to big in the output! Fix like this (or maybe sizes 5 and 7 instead of 6/8? whatever you think looks better!) - 

```suggestion
  theme_pubr() + 
    # Need to make text very small
  theme(
    legend.text = element_text(size = 6),
    legend.title = element_text(size = 8)
  )
```

      y = score) +
  # remove outliers
  geom_boxplot(outlier.shape = NA, color = "grey40", size = 0.2) + 

>   geom_jitter(shape = 1, width = 0.15, size = 0.75, alpha = 0.6) + 

Reviewer: Since this plot appears next to the one we added the legend too, I think we can use the same cancer group colors here (`color = cancer_group_hex`), and revert back to default shapes (no `shape=1` argument). We can indicate in the caption that colors are same as in adjacent panel.
